### PR TITLE
New range() function support for character ranges

### PR DESCRIPTION
**Pull request by joshcom**

### CODE REVIEW

File: lib/sparkql/function_resolver.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-require 'time'
+require 'time'
```
Comment (baron-hines) [inline lib/sparkql/function_resolver.rb]: Nitpick, trailing whitespace.

**Comment by baron-hines** [comment]
:+1:

File: lib/sparkql/function_resolver.rb [hunk 2]
```diff
@@ -344,6 +348,14 @@ def radius(coords, length)
     }
   end
   
+  def range(start_str, end_str)
+    {
+      :type => :character,
+      :operator => "Bt",
```
Comment (wlmcewen) [inline lib/sparkql/function_resolver.rb]: Smart.

**Comment by wlmcewen** [comment]
Any thoughts on range requirements? e.g. perhaps validation should fail for something like:

    MapCoordinates Eq range('M04','M01')

It doesn't quiet sound like a complete solution, I know. Joy has found saved searches like this where the ranges guaranteed there will never be a result. It's just a matter of where we inform the user, and whether we should hit the db.

**Comment by wlmcewen** [comment]
Oh right, and :shipit:

**Pull request merged**